Lib/ufoLib/glifLib.py

```diff
@@ -1292,13 +1290,11 @@ def _number(s):
 	GlifLibError: Could not convert a to an int or float.
 	"""
 	try:
-		n = int(s)
-		return n
+		return int(s)
 	except ValueError:
 		pass
 	try:
-		n = float(s)
-		return n
+		return float(s)
```

Comment:
changes like these make no difference at all, believe me. Stylistically I may agree with you, but it's not gonna make any measurable difference.

Reply:
haha I don't believe otherwise, I made a lot of changes tot he file which I reverted in part, I'll remove this